### ISSUE TITLE
Insufficient color contrast for error-label in login page

### ISSUE THREAD

**Issue by chmst** [joomla/backend-template#27]
### What needs to be fixed
In the login page the label for error has not sufficient color contrast. Try to login without password and you can see it.

### Why this should be fixed
Information is nearly invisible also for sighted persons

### How would you fix it
Define a color for the error message with contrast-ratio at least 4.5 in CSS

### Side Effects expected
These labes can occur on different forms (i.e. contact)
